### DISCUSSION TITLE
Как выбирать технологии под проект?

### DISCUSSION THREAD

**Discussion by dmitriy-uvin** [HowProgrammingWorks/LiveQA#23] [Q&A]
Как выбирать технологии под проект, то что лучше знаешь, а потом переписывать под более подходящую?
с 0 имеется ввиду

**Comment by tshemsedinov** [comment]
- Сейчас почти каждый технологический стек настолько универсален, что может решить почти любую задачу
- Исходить нужно из того, какие у вас есть знания на текущий момент или что вы хотите изучить (если есть цель учить)
- Если есть команта, то нужно исходить из того, к чему команда привыкла и будет более эффективна

Ответ в видео тут https://youtu.be/DFBm4cNYaAY?t=1008
